src/Sarif/CodeGenHints.json

```diff
@@ -34,6 +34,11 @@
       "$type": "Microsoft.Json.Schema.ToDotNet.DictionaryHint, Microsoft.Json.Schema.ToDotNet"
     }
   ],
+  "RunInfo.FileInfo": [
+    {
+      "$type": "Microsoft.Json.Schema.ToDotNet.DictionaryHint, Microsoft.Json.Schema.ToDotNet"
+    }
+  ],
```

Comment:
This code generation hint tells Json.Schema.ToDotNet to emit an `IDictionary` for the `RunInfo.FileInfo` property. Without the hint, it would emit an `object` (because the JSON schema type for this property is `object`).